sass/themes/cr/2017/components/typography/typography.twig

```diff
@@ -1,36 +1,114 @@
+  <span class="breakpoint-sm"><b>Current breakpoint: SM </b><br/> font-size: 20px / line-height: 31px</span>
+</p>
+<br/>
+<br/>
+<br/>
+<h3>Paragraphs</h3>
+<p class="font--xlarge">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+<p><b>.font--xlarge</b><br/>
+  <span class="breakpoint-lg"><b>Current breakpoint: LG </b><br/> font-size: 23px / line-height: 33px</span>
+  <span class="breakpoint-md"><b>Current breakpoint: MD </b><br/> font-size: 23px / line-height: 28px</span>
+  <span class="breakpoint-sm"><b>Current breakpoint: SM </b><br/> font-size: 23px / line-height: 28px</span>
+</p>
+<p class="font--large">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+<p><b>.font--large</b><br/>
+  <span class="breakpoint-lg"><b>Current breakpoint: LG </b><br/> font-size: 20px / line-height: 31px</span>
+  <span class="breakpoint-md"><b>Current breakpoint: MD </b><br/> font-size: 20px / line-height: 31px</span>
+  <span class="breakpoint-sm"><b>Current breakpoint: SM </b><br/> font-size: 20px / line-height: 31px</span>
+</p>
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+<p><span class="breakpoint-lg"><b>Current breakpoint: LG </b><br/> font-size: 18px / line-height: 29px</span>
+  <span class="breakpoint-md"><b>Current breakpoint: MD </b><br/> font-size: 18px / line-height: 29px</span>
+  <span class="breakpoint-sm"><b>Current breakpoint: SM </b><br/> font-size: 18px / line-height: 29px</span> 
+</p>
+<p class="font--small">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+<p class="font--small"><b>.font--small</b><br/>
+<span class="breakpoint-lg"><b>Current breakpoint: LG </b><br/> font-size: 16px / line-height: 27px</span>
+  <span class="breakpoint-md"><b>Current breakpoint: MD </b><br/> font-size: 16px / line-height: 27px</span>
+  <span class="breakpoint-sm"><b>Current breakpoint: SM </b><br/> font-size: 16px / line-height: 27px</span> 
+</p>
 
-<h1>This is fun, right?</h1>
-<p>h1 (L): Used mainly for headers on landing pages</p>
-<h1 class="font--large">This is fun, right?</h1>
-<p>h1: Used in secondary page headers</p>
-<h2>This is fun, right?</h2>
-<p>h2: Used within pages, normally as row headings</p>
-<h3>This is fun, right?</h3>
-<p>h3: Used within pages, normally as row headings</p>
-<h4>This is fun, right?</h4>
-<p>h4: Used within pages, normally as row headings</p>
-<h5>This is fun, right?</h5>
-<p>h5: Used within pages, normally as row headings</p>
-<br/>
-<br/>
-<br/>
-<h4>Paragraphs</h4>
+<p class="font--xsmall">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+<p class="font--xsmall"><b>.font--xsmall</b><br/>
+<span class="breakpoint-lg"><b>Current breakpoint: LG </b><br/> font-size: 14px / line-height: 26px</span>
+  <span class="breakpoint-md"><b>Current breakpoint: MD </b><br/> font-size: 14px / line-height: 26px</span>
+  <span class="breakpoint-sm"><b>Current breakpoint: SM </b><br/> font-size: 14px / line-height: 26px</span>
+</p>
```

Comment:
think you forgot the lorum ipsum here.

Reply:
I can't see it look fine to me

Reply:
Underneath the `.font--xsmall` example there's no lorum ipsum example. Just mentioned it for consistency.
<img width="575" alt="screen shot 2018-10-19 at 14 56 39" src="https://user-images.githubusercontent.com/21171954/47222682-47138800-d3af-11e8-943e-41f96a830034.png">

Reply:
oh sorry my bad... is the example above it? 
Perhaps we should make alternating background colours (like looking at a table) for the examples. I do find it confusing.

Reply:
Probably as it is really for their benefit than ours